selene-lib/default_std/roblox_base.yml

```diff
@@ -107,6 +107,17 @@ globals:
     must_use: true
   CFrame.identity:
     property: read-only
+  CFrame.lookAlong:
+    args:
+      - type:
+          display: Vector3
+      - type:
+          display: Vector3
+      - required: false
+      - type:
+          display: Vector3
+    must_use: true
+
```

Comment:
```suggestion
```

Reply:
Resolved in: https://github.com/Kampfkarren/selene/pull/582/commits/83e27fdf34dc2400caf2cfa5828621a6b10e985d